src/hooks/index.tsx

```diff
@@ -80,9 +78,7 @@ export async function getFileContent(
     apiUrl,
     PAT
       ? {
-          headers: {
-            Accept: `Bearer ${PAT}`,
-          },
+          headers: PAT ? { Authorization: `token ${PAT}` } : {},
```

Comment:
I don't think this one is going to work, on `raw.githubusercontent.com` the `Authorization` header is blocked by CORS. or at least that is what I recall from doing this on `blocks-template`.